source/claim.ts

```diff
@@ -18,7 +18,13 @@ export interface ClaimSnak {
 	readonly snaktype: string;
 }
 
-export type ClaimSnakValue = ClaimSnakTimeValue | ClaimSnakEntityValue
+export type ClaimSnakValue = ClaimSnakStringValue | ClaimSnakTimeValue | ClaimSnakEntityValue | ClaimSnakQuantityValue
+
```

Comment:
Improve formatting to be easier extensible for more additions here.

I couldn't push these nitpicks myself to this pr. Maybe the "allow edits by maintainers" tick isnt set for this PR?

```suggestion
export type ClaimSnakValue =
       ClaimSnakEntityValue |
       ClaimSnakQuantityValue |
       ClaimSnakStringValue |
       ClaimSnakTimeValue
```